File: Topic Based Problems/01 Fundamentals/Beginner/20 Inches to Centimeters/C/InchestoCentimeters.c
/**
 * @file InchestoCentimeters.c
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Inches to Centimeters in C
 * @version 1.0
 * @date 2023-09-24
 * 
 * Copyright (c) 2023
 */

#include <stdio.h>

int main() {
    double inches;

    printf("Enter the length in Inches: ");
    scanf("%lf", &inches);

    printf("Length in Centimeters = %.2lf\n", inches * 2.54);

    return 0;
}

File: Topic Based Problems/01 Fundamentals/01 Beginner/20 Inches to Centimeters/Java/InchestoCentimeters.java
/**
 * @file InchestoCentimeters.java
 * @author SharifdotG
 * @brief Inches to Centimeters in Java
 * @version 1.0
 * @date 2023-09-24
 * 
 * Copyright (c) 2023
 */

import java.util.Scanner;

public class InchestoCentimeters {
    public static void main(String[] args) {
        double inches;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the length in Inches: ");
        inches = scanner.nextDouble();

        System.out.printf("Length in Centimeters = %.2f%n", inches * 2.54);

        scanner.close();
    }
}
